lk2nd/device/keys.c

```diff
@@ -77,6 +79,23 @@ int lk2nd_keys_pressed(uint32_t keycode)
 	return 0;
 }
 
+static uint16_t published_keys[] = {
+	KEY_VOLUMEUP,
+	KEY_VOLUMEDOWN,
+	KEY_POWER,
+	KEY_HOME,
+};
+
+static void lk2nd_keys_publish()
```

Comment:
always put (void) in C function declarations without arguments. `()` means "unspecified number of arguments", see e.g. https://stackoverflow.com/questions/51032/is-there-a-difference-between-foovoid-and-foo-in-c-or-c